templates/launchd.plist

```diff
@@ -11,6 +11,9 @@
     # Set the user to run buildbox-agent as (usually `whoami`). It should be a full OS X user created via System Prefs, and the one you installed buildbox-agent under.
     sudo sed -i '' "s/your-build-user/[insert username here]/g" /Library/LaunchDaemons/io.buildbox.buildbox-agent.plist
 
+    # Set the agent's name
+    sudo sed -i '' "s/your-agent-name/[insert agent name]/g" /Library/LaunchDaemons/io.buildbox.buildbox-agent.plist
+
```

Comment:
@tommeier can you change the destination folder for the plist to `/Library/LaunchAgents` as discussed. Moving it to that location solved my fork exec errors on system reboot.